prprocessor/config/repos.yaml

```diff
@@ -73,7 +73,11 @@ theforeman/foreman-packaging:
 theforeman/foreman_remote_execution:
   redmine: foreman_remote_execution
   redmine_required: true
-  redmine_version_prefix: "foreman_remote_execution "
+  redmine_version_prefix: "foreman_remote_execution-"
+theforeman/katello-pull-transport-migrate:
+  redmine: foreman_remote_execution
+  redmine_required: true
+  redmine_version_prefix: "katello-pull-transport-migrate "
```

Comment:
Are you sure you'll have a space here? Looks like the rest uses a dash as a convention.